.appveyor.yml

```diff
@@ -52,18 +52,18 @@ install:
   - echo extension_dir=ext>>php.ini
   - echo extension=php_mbstring.dll>>php.ini
   - echo extension=php_openssl.dll>>php.ini
-  - cd %APPVEYOR_BUILD_FOLDER%
+  - cd "%APPVEYOR_BUILD_FOLDER%"
```

Comment:
Do you know why this is required now?

Reply:
This particular instance is not required, but it was required for `%ANDROID_HOME%` which now point to the pre-installed version in a path with spaces (also see the next commit). But instead of only quoting `%ANDROID_HOME%` I wanted to consistently quote all paths.

PS: I need to do one more fixup for a README change after I see the AppVeyor build passing once.

Reply:
If you have to push again anyway, you could add above explanation to the commit message.

Reply:
Done.